specification/dns/resource-manager/Microsoft.Network/preview/2015-05-04-preview/dns.json

```diff
@@ -118,6 +118,9 @@
               "$ref": "#/definitions/RecordSet"
             }
           },
+          "201": {
+            "description": "RecordSet was created"
```

Comment:
Do we need schema ref here like we have for 200? We return the recordset body in both cases.

Reply:
I think 200 only required schema. Let me check.

Reply:
Fixed this.